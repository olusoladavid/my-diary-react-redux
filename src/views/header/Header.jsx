import React from "react";
import { Link } from "react-router-dom";

import SideBar from "../../containers/sidebar/SideBar.jsx";

const Header = () => {
  return (
    <header className="header">
      <div className="container">
        <div className="header__brand">
          <Link className="link" to="/">
            MyDiary
          </Link>
        </div>
        <SideBar />
      </div>
    </header>
  );
};

export default Header;
