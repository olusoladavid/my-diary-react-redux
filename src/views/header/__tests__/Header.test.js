import Header from "../Header.jsx";

it("should render without crashing", () => {
  const wrapper = shallow(<Header />);
  expect(wrapper.exists()).toBe(true);
});
