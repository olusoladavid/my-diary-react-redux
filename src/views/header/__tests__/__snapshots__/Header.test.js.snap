// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render without crashing 1`] = `
<header
  className="header"
>
  <div
    className="container"
  >
    <div
      className="header__brand"
    >
      <Link
        className="link"
        replace={false}
        to="/"
      >
        MyDiary
      </Link>
    </div>
    <withRouter(Connect(SideBar)) />
  </div>
</header>
`;
